### PR TITLE
Docs: Progress fix + proposal

### PR DESCRIPTION
**Pull request by louismaximepiton**
### Description

Tweak a bit the wording and remove some space.
Proposal to use aria-valuemax better inside multiple bars.

### Motivation & Context

Better logic for documentation.

### Type of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (non-breaking change)
- [ ] Breaking change (fix or feature that would change existing functionality)

### Checklist

- [x] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)
- [x] My code follows the code style of the project _(using `npm run lint`)_
- [x] My change introduces changes to the documentation
- [x] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed

#### Live previews

- <https://deploy-preview-38014--twbs-bootstrap.netlify.app/docs/5.3/components/progess#multiple-bars>

### Related issues

NA

### CODE REVIEW

File: site/content/docs/5.3/components/progress.md [hunk 1]
```diff
@@ -137,13 +137,13 @@ You can include multiple progress components inside a container with `.progress-
...
-  <div class="progress" role="progressbar" aria-label="Segment one" aria-valuenow="15" aria-valuemin="0" aria-valuemax="100" style="width: 15%">
+  <div class="progress" role="progressbar" aria-label="Segment one" aria-valuenow="15" aria-valuemin="0" aria-valuemax="20" style="width: 15%">
     <div class="progress-bar"></div>
   </div>
-  <div class="progress" role="progressbar" aria-label="Segment two" aria-valuenow="30" aria-valuemin="0" aria-valuemax="100" style="width: 30%">
+  <div class="progress" role="progressbar" aria-label="Segment two" aria-valuenow="30" aria-valuemin="0" aria-valuemax="50" style="width: 30%">
     <div class="progress-bar bg-success"></div>
   </div>
-  <div class="progress" role="progressbar" aria-label="Segment three" aria-valuenow="20" aria-valuemin="0" aria-valuemax="100" style="width: 20%">
+  <div class="progress" role="progressbar" aria-label="Segment three" aria-valuenow="20" aria-valuemin="0" aria-valuemax="30" style="width: 20%">
```
Comment (louismaximepiton) [inline site/content/docs/5.3/components/progress.md:146]: Those values could also be 15, 30, 55 or whatever but the sum should be 100 since the `aria-valuenow` equals the width percentage.

Comment (patrickhlauke) [reply site/content/docs/5.3/components/progress.md:146]: reminded of this discussion from the past https://github.com/twbs/bootstrap/issues/32973#issuecomment-775517915

tl;dr: there's no real concept of interdependent progress bars that together form part of a whole in ARIA, and it depends really how an author is using these (either as literally a visual smushing together of various progress bars that go from 0-100%, or segments of something that totals 100% ... both of which I could envisage as fine in certain circumstances)

File: site/content/docs/5.3/components/progress.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ toc: true
 Progress components are built with two HTML elements, some CSS to set the width, and a few attributes. We don't use [the HTML5 `<progress>` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress), ensuring you can stack progress bars, animate them, and place text labels over them.
 
 - We use the `.progress` as a wrapper to indicate the max value of the progress bar.
-- The `.progress` wrapper also requires a `role="progress"` and `aria` attributes to make it accessible, including an accessible name (using `aria-label`, `aria-labelledby`, or similar).
```
Comment (patrickhlauke) [inline site/content/docs/5.3/components/progress.md:18]: oops, good spot

**Comment by patrickhlauke** [comment]
personally, i'm ok with all changes except for the valuemax changes (unless we expand this much further to explain the two possible ways to look at these stacked progress bars, e.g. three separate bars that go 0-100% but that have been visually smushed together, OR separate segments that form part of a whole)

File: site/content/docs/5.3/components/progress.md (hunk 1 above)
Comment (louismaximepiton) [reply site/content/docs/5.3/components/progress.md:146]: Oh yeah I haven't made enough searchs to find this! Thanks a lot to pin this PR. I better understand the context and the actual implementation, reverting this for sure.

**Review by mdo — APPROVED** [review]

**Comment by patrickhlauke** [comment]
party time :party:
